modules/selenium/src/main/java/org/testcontainers/containers/DefaultRecordingFileFactory.java

```diff
@@ -0,0 +1,24 @@
+package org.testcontainers.containers;
+
+import org.junit.runner.Description;
+
+import java.io.File;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class DefaultRecordingFileFactory implements RecordingFileFactory {
+
+    private static final SimpleDateFormat filenameDateFormat = new SimpleDateFormat("YYYYMMdd-HHmmss");
+
+    @Override
+    public File recordingFileForTest(File vncRecordingDirectory, Description description, boolean succeeded) {
+        final String prefix = succeeded ? "PASSED" : "FAILED";
```

Comment:
I think we should put those Strings into constants.

Reply:
Sure.